app/controllers/concerns/sufia/depositors_controller_behavior.rb

```diff
@@ -24,6 +24,7 @@ def create
         head :ok
       else
         grantor.can_receive_deposits_from << grantee
+        send_proxy_depositor_added_messages(grantor, grantee)
```

Comment:
Nm, I'd say. I don't think we want to conflate user messaging with event jobs. Thanks for the PR!